docs/src/main/sphinx/release/release-372.md

```diff
@@ -0,0 +1,137 @@
+  credentials with `user-credential-name` or `password-credential-name` are used
+  to access data. ({issue}`11068`)
+
+## Hive connector
+
+* Allow redirects of Iceberg or Delta tables which have no ``StorageDescriptor``
+  in the Glue metastore. ({issue}`11092`)
+* Stop logging spurious failures when [storage caching](/connector/hive-caching)
+  is used. ({issue}`11101`)
+* Allow reading Zstandard compressed Avro files. ({issue}`11090`)
+* Fix incorrect query results after writes to a table with caching directory
+  lister enabled with the `hive.file-status-cache-tables` configuration
+  property. ({issue}`10621`)
+* Fix potential query failures for queries writing data to tables backed by S3.
+  ({issue}`11089`)
+
+## Iceberg connector
+
+* Add support for ``COMMENT ON COLUMN`` statement. ({issue}`11143`)
+* Improve query performance after table schema evolved, by fixing the connector
+  to support table stats in such case. ({issue}`11091`)
+* Fix potential query failures for queries writing data to tables backed by S3.
+  ({issue}`11089`)
+* Fix dereference pushdown when column has a comment. ({issue}`11104`)
+
+## Kudu connector
+
+* Add support for Kerberos authentication. ({issue}`10953`)
+
+## MongoDB connector
+
+* Infer MongoDB `bindata` type as Trino `varbinary` type if explicit schema does
+  not exist. ({issue}`11122`)
+
+## MySQL connector
+
+* Fix spurious query failures when metadata cache is not enabled, and extra
+  credentials with `user-credential-name` or `password-credential-name` are used
+  to access data. ({issue}`11068`)
+
+## Oracle connector
+
+* Fix spurious query failures when metadata cache is not enabled, and extra
+  credentials with `user-credential-name` or `password-credential-name` are used
+  to access data. ({issue}`11068`)
+
+## Phoenix connector
+
+* Fix query failures when applying predicates on `array(char)` type columns.
+  ({issue}`10451`)
+* Fix metadata listing failure in case of concurrent table deletion.
+  ({issue}`10904`)
+
+## PostgreSQL connector
+
+* Add support for pushing down joins on character string type columns.
+  ({issue}`10059`)
+* Fix spurious query failures when metadata cache is not enabled, and extra
+  credentials with `user-credential-name` or `password-credential-name` are used
+  to access data. ({issue}`11068`)
+
+## Redshift connector
+
+* Fix spurious query failures when metadata cache is not enabled, and extra
+  credentials with `user-credential-name` or `password-credential-name` are used
+  to access data. ({issue}`11068`)
+
+## SingleStore (MemSQL) connector
+
+* Fix spurious query failures when metadata cache is not enabled, and extra
+  credentials with `user-credential-name` or `password-credential-name` are used
+  to access data. ({issue}`11068`)
+
+## SQL Server connector
+
+* Updated JDBC driver to 10.2.0. The new version automatically enables TLS and
+  certificate verification. Update the [TLS configuration](sqlserver-tls) to
```

Comment:
Fyi @hashhar .. linking to updated docs instead of repeating part of the info only.

Reply:
IMO release notes should be up-front about the "fix" for a breaking change.

Reply:
I don't think we should replicate documentation partially if we can just link to it.